### DISCUSSION TITLE
Camera x,y,z controls

### DISCUSSION THREAD

**Discussion by Myztklkev** [praydog/REFramework#525] [General Questions]
any chance you could separate the camera x,y,z controls from the first person mod so they can be used without first person? I like to play with the camera a little tighter, and usually I can do this with a trainer but there is no easy way to install the trainer on the steam deck. The camera controls are already there, they just can't be used without first person enabled.

**Comment by praydog** [comment]
If you're talking about a tighter third person camera, you can do this with the First Person mod. You just need to move the Z offset backwards a bit, you can input any value you want by pressing the CTRL button on the slider. Then you can attach the camera to any bone you want, depending on how much you want the camera to shake based on character movement.

**Reply by Myztklkev** [reply]
I just tried it with first person enabled (I didn't know ctrl let you input any value) but wasn't able to get the result I was looking for. the camera clips into walls and stuff when you do it this way and depending on where you set the camera, you can't see where you're aiming. Also, I don't think you would be able to enter any value you want on the steam deck because the virtual keyboard kinda blows. something similar to the camera controls in the ultimate trainer would be awesome if you ever find the time. thanks for the quick response though.
